frontend/src/router/guards.js

```diff
@@ -115,7 +114,8 @@ function ensureDataLoaded (store, localStorage) {
         case 'NewShoot':
         case 'NewShootEditor': {
           const promises = [
-            store.dispatch('subscribeShoots')
+            store.dispatch('subscribeShoots'),
+            store.dispatch('fetchNetworkingTypes')
```

Comment:
both needed also for the new shoot editor?

Reply:
Yes, we need this to set the initial network type also for the editor (when accessing the editor page directly, without navigating from UI component view)

Reply:
but when you switch from the `NewShootEditor` to the `NewShoot`, this code will be executed again, or not? Then you can only fetch it in case of `NewShoot`

Reply:
If the user enters the route with the editor, the network types must also be loaded. so how to achieve this then?